agentops/client.py

```diff
@@ -264,11 +266,13 @@ def end_session(self,
 
         self._session.video = video
         self._session.end_session(end_state, end_state_reason)
-        token_cost = self._worker.end_session(self._session)
+        getcontext().prec = 6
+        token_cost = Decimal(self._worker.end_session(self._session))
         if token_cost == 'unknown':
             print('🖇 AgentOps: Could not determine cost of run.')
         else:
-            print('🖇 AgentOps: This run cost ${:.6f}'.format(float(token_cost)))
+
+            print('🖇 AgentOps: This run cost ${}'.format('{:.2f}'.format(token_cost) if token_cost == 0 else '{:.6f}'.format(token_cost)))
```

Comment:
I'm going back on what I said earlier, but Decimal isn't actually required here. I ran some experiments to confirm

```
In [16]: f"{0.01:.2f}"
Out[16]: '0.01'
```

Better f-string formatting solves this